Index: speed-dreams-2/data/data/sky/CMakeLists.txt
===================================================================
--- speed-dreams-2/data/data/sky/CMakeLists.txt	(revision 6401)
+++ speed-dreams-2/data/data/sky/CMakeLists.txt	(working copy)
@@ -7,3 +7,4 @@
 FILE(GLOB dds *.dds)
 
 SD_INSTALL_FILES(DATA data/sky FILES ${TXT_FILES})
+SD_INSTALL_FILES(DATA data/sky FILES ${rgb} ${rgba} ${png} ${dds})
