--- a/src/modules/graphic/osggraph/Utils/OsgAtomic.h	2021-10-12 12:52:28.637228572 -0400
+++ b/src/modules/graphic/osggraph/Utils/OsgAtomic.h	2021-10-12 12:53:18.636632675 -0400
@@ -20,7 +20,7 @@
 #ifndef OsgAtomic_H
 #define OsgAtomic_H
 
-#if defined(__GNUC__) && ((4 < __GNUC__)||(4 == __GNUC__ && 1 <= __GNUC_MINOR__)) && (defined(__x86_64__) || defined(__powerpc64__))
+#if defined(__GNUC__) && ((4 < __GNUC__)||(4 == __GNUC__ && 1 <= __GNUC_MINOR__)) && (defined(__x86_64__) || defined(__powerpc64__) || defined(__aarch64__))
 // No need to include something. Is a Compiler API ...
 # define SDATOMIC_USE_GCC4_BUILTINS
 #elif defined(__GNUC__) && (defined(__i386__) || defined(__arm__))
